core/wallet/tron_test.go

```diff
@@ -32,28 +33,28 @@ func TestTransferBTT(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	cfg.Services.SolidityDomain = "grpc.trongrid.io:50052"
-	cfg.Services.FullnodeDomain = "grpc.trongrid.io:50051"
+	cfg.Services.SolidityDomain = "grpc.shasta.trongrid.io:50052"
+	cfg.Services.FullnodeDomain = "grpc.shasta.trongrid.io:50051"
 	cfg.Services.EscrowDomain = "https://escrow-staging.btfs.io"
-	// don't have dev token now, will delete this line later
-	TokenIdDev = "1002000"
-	privKey, err := crypto.ToPrivKey("CAISID6wbQHX3IHneklC2rOMyH250Wijf5W7phqPHZ1P5fFC")
+	TokenIdDev = "1000252"
+	privKey, err := crypto.ToPrivKey("CAISILOZbORDZlczUlp5jdonb5y5SMZgaZy6OWp58SkS8jS8")
 	if err != nil {
 		t.Fatal(err)
 	}
-	ret, err := TransferBTT(context.Background(), node, cfg, privKey, "41A2468828043EE28B2C2556E063DAD6A314372E2F",
+	ret, err := TransferBTT(context.Background(), node, cfg, privKey, "41BC8E7F3E2BB11310B75D6B0B6E8537D069CDB72E",
 		"416E2FFC26BDF48B1983CCC9EC2521867F98667760", 1)
 	if err != nil {
 		t.Fatal(err)
 	}
 	assert.True(t, ret.Result)
 	assert.Equal(t, "SUCCESS", ret.Code)
 
-	ret2, err := TransferBTT(context.Background(), node, cfg, privKey, "TQmExmuk8KyUByiS36c5sAfke8PghojhDo",
+	ret2, err := TransferBTT(context.Background(), node, cfg, privKey, "TTACjzSeJ9jDHaxRxnho1n3mVK9JASNyr9",
 		"TL1ppDNyESQ5msZ1BBZQEFg6ksdbcyWDwj", 1)
 	if err != nil {
 		t.Fatal(err)
 	}
 	assert.True(t, ret.Result)
 	assert.Equal(t, "SUCCESS", ret2.Code)
+	time.Sleep(80 * time.Second)
```

Comment:
Why sleep 80s then check nothing? You need to check confirmed state somehow right? If not, then no need to sleep.